adapters/NAS/AzureNetAppFiles/README.md

```diff
@@ -0,0 +1,75 @@
+# Protect Azure NetApp Files with Cohesity Generic NAS Pre & Post Script
+
+## Overview
+
+Cohesity custom pre & post script for Azure NetApp Files, allows you to backup specific snapshots of Azure NetApp Files to Cohesity storage.
+
+### Prerequisites
+
+Below is the list of prerequisites which should meet before configuring the Protection Group:
+
+1. Ensure that you have a Linux Control VM configured and have access to Azure NetApp Files volumes
+
+
+2. Prepare SSH access and scripts on Linux VM:
+
+``` bash
+# Log in to the Linux VM and enter the command 
+cd .ssh 
+
+```
+  * Copy the SSH key from Protection Group Pre & Post Scripts section and Paste the copied SSH Key in the authorized_keys file.
+``` bash
+cat >> authorized_keys
+# Press Contrl+D to save the file
+
+```
+  * Create a new directory called scripts in the Linux VM and create new files in it
+``` bash 
+mkdir scripts
+
+```
+  * Download the scripts from the current folder
+
+  ``` bash
+  # Copy create_anf_snapshot.ps1 content and paste in the file create_anf_snapshot.ps1
+
+cat > create_anf_snapshot.ps1
+
+# Press Contrl+D to save the file
+
+   # Copy delete_anf_snapshot.ps1 content and paste in the file delete_anf_snapshot.ps1
+
+   cat > delete_anf_snapshot.ps1
```

Comment:
I'd just say copy the file from the repository to the powershell VM

Reply:
done